src/Hit/Issue.hs

```diff
@@ -90,6 +92,22 @@ showIssueFull i@Issue{..} = T.intercalate "\n" $
     highlight :: Text -> Text
     highlight x = boldCode <> greenCode <> x <> resetCode
 
+-- | Create an 'Issue' by given 'Text'
+createIssue' :: Text -> IO (Either Error Issue)
+createIssue' title = getOwnerRepo >>= \case
+    Just (owner, repo) -> do
+        token <- lookupEnv "GITHUB_TOKEN"
+        let gitHubToken = OAuth . encodeUtf8 <$> token
+        case gitHubToken of
+            Just oAuth -> createIssue oAuth owner repo (newIssue title)
+            Nothing -> do
+                let errTxt = "Can not get GITHUB_TOKEN"
+                errorMessage errTxt
+                pure $ Left $ ParseError errTxt
+    Nothing -> do
+        errorMessage noOwnerRepoError
+        pure $ Left $ ParseError noOwnerRepoError
```

Comment:
There's some code duplication between this function and `withOwnerRepo`

Reply:
Should be removed once the implemention for above discussion is done :wink: